cypress/support/commands.js

```diff
@@ -337,8 +337,9 @@ Cypress.Commands.add('clearContent', () => {
 })
 
 Cypress.Commands.add('openWorkspace', () => {
-	cy.get('#rich-workspace .empty-workspace').click()
-	cy.getEditor().find('[data-text-el="editor-content-wrapper"]').click()
+	cy.createDescription()
+	cy.get('#rich-workspace .editor__content').click({force: true})
```

Comment:
This looks like the right place to wait until the editor content is visible. instead of forcing actions all over the place ;)